src/lib/Behat/Component/Table/TableRow.php

```diff
@@ -97,6 +113,8 @@ protected function specifyLocators(): array
             new VisibleCSSLocator('checkbox', 'input[type=checkbox]'),
             new VisibleCSSLocator('assign', '[data-original-title="Assign content"],[data-original-title="Assign to Users/Groups"]'),
             new VisibleCSSLocator('edit', '.ibexa-icon--edit,[data-original-title="Edit"]'),
+            new VisibleCSSLocator('de-active', '.ibexa-icon--edit,[data-original-title="De-activate"]'),
```

Comment:
```suggestion
            new VisibleCSSLocator('de-active', '.ibexa-icon--edit,[data-original-title="De-activate"]'),
```

I don't think the active/deactive buttons have the pencil editing icon (`.ibexa-icon--edit`), can we remove it from here?